notebooks/05-operations/01-druid-logs-configuration.ipynb

```diff
@@ -0,0 +1,524 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bec19737-2773-4c66-a05e-d0b1a707c4c8",
+   "metadata": {},
+   "source": [
+    "## Appenders\n",
+    "\n",
+    "There are two `appenders`:\n",
+    "\n",
+    "* \"Console\", which is a a [`Console`](https://logging.apache.org/log4j/log4j-2.4/manual/appenders.html#ConsoleAppender) appender for `SYSTEM_OUT`.\n",
+    "* \"FileAppender\", which is a [`RollingRandomAccessFile`](https://logging.apache.org/log4j/log4j-2.4/manual/appenders.html#RollingRandomAccessFileAppender) appender for detailed process logs.\n",
+    "\n",
+    "### Start a Druid instance\n",
+    "\n",
+    "The default Configuration for Druid does not include a [`monitorInterval`](https://logging.apache.org/log4j/log4j-2.4/manual/configuration.html#AutomaticReconfiguration) property, so changes to logging configuration are only recognised when a process restarts.\n",
+    "\n",
+    "For the purposes of this tutorial, run the following command to add a `monitorInterval` property to the Configuration.\n",
+    "\n",
+    "```bash\n",
+    "sed -i '' 's/<Configuration status=\"WARN\">/<Configuration status=\"WARN\" monitorInterval=\"5\">/' \\\n",
+    "  ~/learn-druid-local/apache-druid/conf/druid/auto/_common/log4j2.xml\n",
+    "```\n",
+    "\n",
+    "Start Druid with the following command:\n",
+    "\n",
+    "```bash\n",
+    "nohup ~/learn-druid-local/apache-druid/bin/start-druid & disown > log.out 2> log.err < /dev/null\n",
+    "```\n",
+    "\n",
+    "### Look at the standard log files\n",
+    "\n",
+    "Since Druid is a distributed system, we will find log files for each Druid process. In addition, Druid also captures the output written to the standard output. Use the following command to take a look at what is here:\n",
+    "\n",
+    "```bash\n",
+    "cd ~/learn-druid-local/apache-druid/log ; ls\n",
+    "```\n",
+    "\n",
+    "You can see how this results in two sets of files being created:\n",
+    "\n",
+    "* `process name.stdout.log` file - which is the information written by the processes to stdout (i.e., the terminal)\n",
+    "* `process name.log` - file containing various status, error, warning and debug messages"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5258912e-bf23-4acd-9267-a7f4ea3bc6f4",
+   "metadata": {},
+   "source": [
+    "### Log filenames\n",
+    "\n",
+    "The \"FileAppender\" `RollingRandomAccessFile` appender has both `fileName` and `filePattern` properties. The `fileName` property of the \"FileAppender\" sets the name of the log file being written to at the moment, while `filePattern` is applied when the log rolls over.\n",
+    "\n",
+    "Run the following command to change the default filename of the \"FileAppender\" in the log4j configuration file so that all future detailed process logs will have a name suffixed with the hostname.\n",
```

Comment:
```suggestion
    "Run the following command to change the default filename of the \"FileAppender\" in the Log4j configuration file so that all future detailed process logs will have a name suffixed with the hostname.\n",
```